pages/servers.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title:  "Servers"
+permalink: /servers/
+---
+
+List of servers (in alphabetical order) that provide a CCTray feed:
+
+### Buddybuild
+
+* Server home: <https://www.buddybuild.com/>
+* CCMenu setup instructions: <http://docs.buddybuild.com/docs/ccmenu>
+
+### CircleCI
+
+* Server home: <https://circleci.com/>
+* CCMenu setup instructions: <https://circleci.com/docs/polling-project-status>
+
+### CruiseControl
+
+* Server home: <http://cruisecontrol.sourceforge.net/>
+* Default feed location (classic): `/xml`
+* Default feed location (dashboard): `/cctray.xml` or `/dashboard/cctray.xml`
+* CCMenu autodetect feed URL: supported
+
+### CruiseControl.NET
+
+* Server home: <http://www.cruisecontrolnet.org/>
```

Comment:
This site no longer exists

Reply:
https://ccnet.github.io/CruiseControl.NET/projects/ccnet/wiki.html

Reply:
Will fix the link.